recipes-security/selinux/files/selinux-autorelabel.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
Why do you need to place this file if you use standard one?

Reply:
This file was used for testing. Remove it and used default from another recipe